docs/DeveloperGuide.md

```diff
@@ -308,7 +308,64 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case resumes at step 2.
 
-*{More to be added}*
+
+**Use case 2: Delete a project**
+
+**MSS**
+
+1.  User requests to list projects
```

Comment:
Not too sure myself but, is the use case supposed to start from the listing of projects/employees or does it start from the action of deleting itself?